tracer/test/Datadog.Trace.Tests/MultipartFormTests.cs

```diff
@@ -0,0 +1,90 @@
+// <copyright file="MultipartFormTests.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System;
+using System.IO;
+using System.Text;
+using System.Threading.Tasks;
+using Datadog.Trace.Agent;
+using Datadog.Trace.Agent.Transports;
+using Datadog.Trace.TestHelpers;
+using VerifyTests;
+using VerifyXunit;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Datadog.Trace.Tests
+{
+    [Collection(nameof(WebRequestCollection))]
+    [UsesVerify]
+    public class MultipartFormTests : TestHelper
+    {
+        public MultipartFormTests(ITestOutputHelper output)
+            : base(string.Empty, output)
+        {
+            VerifierSettings.DerivePathInfo(
+                (sourceFile, projectDirectory, type, method) =>
+                {
+                    return new(directory: Path.Combine(projectDirectory, "..", "snapshots"));
+                });
+        }
```

Comment:
This is global configuration, we shouldn't be calling it in more than one place 🤔 The `VerifyHelper` should handle this

Reply:
`VerifyHelper` is not available in the `Datadog.Trace.Tests` project, that's why I'm calling this manually.

Reply:
You can _make_ it available like this 🙂 

https://github.com/DataDog/dd-trace-dotnet/blob/611f68d836a26c91b49be30bea9bb0e5ee482bf6/tracer/test/Datadog.Trace.ClrProfiler.IntegrationTests/Datadog.Trace.ClrProfiler.IntegrationTests.csproj#L14

Reply:
done in [379f632](https://github.com/DataDog/dd-trace-dotnet/pull/2962/commits/379f632eb689e5cd66bd53bd7509390b3a96a99b)